dtformats/indx_directory_entry.py

```diff
@@ -0,0 +1,144 @@
+# -*- coding: utf-8 -*-
+"""INDX entries """
```

Comment:
Are these NTFS $I30 index entries? Note that NTFS support different types of index entries as well, so please be as specific as possible here.